### ISSUE TITLE
Rename nonSelfUpdate to follow new filter naming conventions

### ISSUE THREAD

**Issue by dariusf** [HubTurbo/HubTurbo#954]
It should be written as `non-self-update`. The filter docs should be updated as well.

**Comment by damithc** [comment]
I prefer to limit to two words if possible. Shouldn't it be `sort:updated-others`? Using the same command here increases consistency but would it be confusing to the user or the parser?

**Comment by dariusf** [comment]
Right, it's 'updated-others'. I don't think it would be confusing.

**Comment by codemanmeet** [comment]
Yup, even I think so. It shouldn't be confusing.
